lib/test/tasks/run_file_size_checks.rb

```diff
@@ -18,8 +18,8 @@ class Test::Tasks::RunFileSizeChecks < Pallets::Task
     'emoji_picker*.css' => (8..18),
     'emoji_picker*.js' => (322..332),
     'google_sign_in_button*.js' => (1..11),
-    'groceries*.css' => (60..70),
-    'groceries*.js' => (319..329),
+    'groceries*.css' => (48..58),
+    'groceries*.js' => (303..313),
```

Comment:
The decreased file sizes here are a small but nice benefit of removing the groceries menu for account management.